src/include/btree_inline.h

```diff
@@ -1401,6 +1401,7 @@ __wt_page_evict_retry(WT_SESSION_IMPL *session, WT_PAGE *page)
     WT_TXN_GLOBAL *txn_global;
     wt_timestamp_t pinned_ts;
 
+    pinned_ts = 0U;
```

Comment:
Variable initializations follow the same order as declarations. So pinned_ts should be assigned a value after txn_global. Assignment should be to "WT_TS_NONE".